src/api/app/views/webui/package/branch.html.haml

```diff
@@ -0,0 +1,28 @@
+:ruby
+  @pagetitle = "Branch #{@package}"
+
+.card.mb-3
```

Comment:
Inside containers you normally use [the grid](https://getbootstrap.com/docs/4.4/layout/grid/) to place things, not cards.

Reply:
@hennevogel I applied the grid system, but I kept the card. We use them almost everywhere. Not sure if I understood correctly, do you want to get rid of the way we use the cards to structure the view or was it more about the missing grid?

Reply:
You now have a wild mixture of card and grid. The grid is for the layout, the card is styling the content. Build a grid, fill it with content.

Reply:
@hennevogel I see that in many PRs we are going with `card.mb-3` and you are against it. Could you please explain us how you want it to avoid everyone doing their own interpretation of what you are saying?

Reply:
we talked about this offline. grid first, then put content onto it is our new motto! :smile: